infrastructure/scripts/provision-infra.sh

```
@@ -0,0 +1,117 @@
+    esac
+}
+
+jq -n \
+    --arg environment "$ENVIRONMENT" \
+    --arg backend_image_tag "$TAG" \
+    --arg frontend_image_tag "$TAG" \
+    --arg django_configuration "$DJANGO_CONFIGURATION" \
+    --arg django_settings_module "$DJANGO_SETTINGS_MODULE" \
+    --arg domain_name "$DOMAIN_NAME" \
+    --argjson enable_cron_tasks "$(_normalize_bool "${ENABLE_CRON_TASKS:-false}")"
+    --arg db_password "$DB_PASSWORD" \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)